### ISSUE TITLE
CAMERA_IDLE works on iOS but not Android; CAMERA_CHANGED functions like CAMERA_IDLE

### ISSUE THREAD

**Issue by caleb87** [mapsplugin/cordova-plugin-googlemaps#1172]
CAMERA_IDLE does nothing on Android

CAMERA_CHANGED functions just like CAMERA_IDLE does on iOS (doesnt fire until panning stops)

on iOS, CAMERA_CHANGED fires constantly as the map is panning, as expected

on iOS, CAMERA_IDLE fires when the map has stopped moving, as expected

**Comment by wf9a5m75** [comment]
https://github.com/mapsplugin/cordova-plugin-googlemaps/wiki/Map#listen-events

```
MAP_CLICK
MAP_LONG_CLICK
MY_LOCATION_BUTTON_CLICK
CAMERA_CHANGE
CAMERA_IDLE(iOS)
MAP_READY
MAP_LOADED(Android)
MAP_WILL_MOVE(iOS)
MAP_CLOSE
```

**Comment by wf9a5m75** [comment]
If you want to catch the `CAMERA_IDLE` on Android, you need to use the v2-beta version.

The v2-beta version supports the `CAMERA_MOVE_START`, `CAMERA_MOVE` and `CAMERA_MOVE_END` events for both Android and iOS.
https://github.com/mapsplugin/cordova-plugin-googlemaps/wiki/Migrate-from-v1-to-v2#version-2x-events

**Comment by caleb87** [comment]
Thanks for the information. 

It's only confusing because CAMERA_CHANGE works exactly the same as CAMERA_IDLE on Android. For Android I see no event similar to CAMERA_CHANGE on iOS. 

To catch CAMERA_IDLE on Android, I'm doing this: 
```
if(device.platform == 'Android') { 
  map.on(plugin.google.maps.event.CAMERA_CHANGE, cameraChanged);
} else { 
  map.on(plugin.google.maps.event.CAMERA_IDLE, cameraChanged);
}
```
This code works exactly the same on both platforms. If I used CAMERA_CHANGE only, then iOS would fire 100 times per second.

**Comment by wf9a5m75** [comment]
I know. When I implemented the event at the very fist version, I didn't understand the CAMERA_IDLE event well.
I noticed this very late, but it's already became a popular this plugin,
so I could not change the event name.

I strongly recommend you use the maps plugin v2-beta.
The v1 will be shutdown in near future.
https://github.com/mapsplugin/v2.0-demo
https://github.com/mapsplugin/cordova-plugin-googlemaps/wiki/v2-beta

Almost the same code work, but the event names and some methods are changed.
